tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -35,11 +36,105 @@
 _doc_root: str = ""
 
 
+class CapsuleType(IntEnum):
+    # Defined in
+    # https://www.ietf.org/archive/id/draft-ietf-masque-h3-datagram-03.html.
+    DATAGRAM = 0xff37a0
+    REGISTER_DATAGRAM_CONTEXT = 0xff37a1
+    REGISTER_DATAGRAM_NO_CONTEXT = 0xff37a2
```

Comment:
> we should also support DATAGRAM

For WebTransport over HTTP/3 we use https://ietf-wg-masque.github.io/draft-ietf-masque-h3-datagram/draft-ietf-masque-h3-datagram.html#name-http-3-datagram-format rather than https://ietf-wg-masque.github.io/draft-ietf-masque-h3-datagram/draft-ietf-masque-h3-datagram.html#name-the-datagram-capsule, right?

Reply:
Chromium accepts both, though I don't believe we send the DATAGRAM capsule currently.